pocs/observatory.py

```diff
@@ -202,6 +209,9 @@ def status(self):
         """
         status = {}
         try:
+            if not self.can_observe:
```

Comment:
We should not be raising an Exception in the status call. We always want to be able to return a status. That status should indicate if observing is possible or not. Basically, `status['can_observe'] = self.can_observe` is all that's needed here.